Environment/solaredge.5m.py

```diff
@@ -10,6 +10,9 @@
 # <xbar.var>string(SITE_ID): Please provide your SolarEdge installation SiteId</xbar.var>
 # <xbar.var>string(API_KEY): Please provide your SolarEdge installation API Key (https://www.solaredge.com/node/88689)</xbar.var>
 # <xbar.var>string(BATTERY_PRESENT): Does your SolarEdge installation include a battery? Y/N</xbar.var>
+# <xbar.var>string(SYSTEM_WATTAGE): Total panel DC watt capacity</xbar.var>
+# <xbar.var>string(SHOW_ENVIRONMENTAL_BENEFIT): Do you want to show your environmental benfit in the dropdown? Y/N (Default Y)</xbar.var>
```

Comment:
Why not a boolean variable here?

Reply:
More readability than anything else - can change to bool if you think it helps.

Reply:
🤷🏽 I'm not a python expert; if you feel this is more readable, that works for me.